### ISSUE TITLE
Input accessibility: Senior official |  Domain request form

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2752]
### Story

Email * field does not communicate expected data formats. An error is not sufficient to meet Section 508 standards.

Email * error state does not include the error message in the screen reader/ ANDI output:
![Screenshot 2024-09-13 at 12 49 44 PM](https://github.com/user-attachments/assets/eacbec05-aab0-4ef7-95d2-5064e4f9b2d9)

### Acceptance Criteria

- [ ] Email * field has visual hint text `Enter an email address in the required format, like name@example.com.`. Match [this design](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=9158-5121&t=xHoOWjASpMD1PN90-4).
- [ ] Email *  error state for an empty field should have this screen reader output: `Who is the senior official for your organization? Email * Enter an email address in the required format, like name@example.com. invalid entry, Enter an email address in the required format, like name@example.com, required.`

### Issue Links

Relates to Request form labels communicate required data format [#2699](https://github.com/cisagov/manage.get.gov/issues/2699)

**Comment by AnnaGingle** [comment]
This issue will be completed as a part of issue #2848
